api/sys_plugins.go

```diff
@@ -36,6 +36,8 @@ type ListPluginsResponse struct {
 type PluginDetails struct {
 	Type              string `json:"type"`
 	Name              string `json:"name"`
+	OCIImage          string `json:"oci_image,omitempty" mapstructure:"oci_image"`
```

Comment:
How come the `mapstructure` tag is needed?

Reply:
https://github.com/hashicorp/vault/blob/3130e8ba9483e10e6191d264dc97b597462bf7c1/api/sys_plugins.go#L113

PluginDetails is parsed using `mapstructure`.

Reply:
Ahh, what I meant to ask was why does this field need it but not the others. But I guess it's because `mapstructure` successfully guesses the correct serialised names for the other fields. Cool 👍 thanks!